test/CreateResourceTest.php

```diff
@@ -26,12 +26,15 @@ public function testReturnsUriOn201()
         $api = new FedoraApi($guzzle);
 
         $result = $api->createResource("");
-        $this->assertSame($result, "SOME URI");
+        $this->assertEquals($result->getHeaderLine("Location"), "SOME URI");
+        $this->assertEquals(201, $result->getStatusCode(), "Expected a 201 response.");
```

Comment:
Or you can do 
```
public function testReturnsUriOn201()
     {
         $mock = new MockHandler([
            new Response(201, ['Location' => "SOME URI"], "SOME URI"),
         ]);
          $handler = HandlerStack::create($mock);
          $guzzle = new Client(['handler' => $handler]);
          $api = new FedoraApi($guzzle);
  
          $result = $api->createResource("");
          $this->assertEquals($result->getHeaderLine("Location"), "SOME URI");
          $this->assertEquals(201, $result->getStatusCode(), "Expected a 201 response.");
          $this->assertEquals("SOME URI", $result->getBody()->getContent());
     }
```

A lot of these tests are because the response was altered by Chullo, now that FedoraApi just returns it straight, they are little more than "return a 201, check that you returned a 201." Which is why they are not super helpful.